website/docs/components/code-editor/partials/guidelines/guidelines.md

```diff
@@ -104,3 +105,22 @@ In **Figma** we provide a handful of example languages intended as visual exampl
 
 If you wish to create custom examples using the Code Editor, we publish all of the relevant syntax highlighting variables in the [HDS Components v2.0](https://www.figma.com/design/iweq3r2Pi8xiJfD9e6lOhF/HDS-Components-v2.0?node-id=67166-37020&t=gWdKy44MzTP4cTRo-1) library. However, due to the number of languages supported by the component, the color variables use a generic naming schema (e.g., cyan, red, purple) to remain as agnostic as possible when being applied to different languages.
 For more details around syntax, visit the [specifications](/components/code-editor?tab=specifications).
+
+## Linting
+The Code Editor supports "linting" (currently only of JSON code) a user's code. This feature will highlight an error with an underline and display an icon next to the line number, each of which will show a tooltip on hover. To view all alerts in the editor, users can open a dialog using Ctrl-Shift-m (Cmd-Shift-m on macOS).
+
+!!! Info 
+Linting within the Code Editor is only supported for JSON. If you require linting for addition languages [contact](/about/support) the HDS team.
+
+This component is only available in Ember and not currently supported in Figma.
```

Comment:
Do we have plans to ever support this in Figma? If not, you might want to consider removing the word "currently" so consumers don't think it's a feature we'll implement in the future. 

```suggestion
This component is only available in Ember and is not supported in Figma.
```